src/main.cpp

```diff
@@ -1242,6 +1245,93 @@ void vcRenderScene_HandlePicking(vcState *pProgramState, vcRenderData &renderDat
     {
       if (pProgramState->udModelPickedIndex < (int)renderData.models.length)
         udStrcpy(pProgramState->sceneExplorer.selectUUIDWhenPossible, renderData.models[pProgramState->udModelPickedIndex]->m_pNode->UUID);
+
+      if (pProgramState->udModelPickedNode != 0)
+      {
+        uint8_t *pAttributePtr = nullptr;
+
+        if (vdkPointCloud_GetAttributeAddress(renderData.models[pProgramState->udModelPickedIndex]->m_pPointCloud, pProgramState->udModelPickedNode, 0, (const void**)&pAttributePtr) == vE_Success)
+        {
+          pProgramState->udModelNodeAttributes.SetVoid();
+
+          vdkPointCloudHeader *pHeader = &renderData.models[pProgramState->udModelPickedIndex]->m_pointCloudHeader;
+
+          for (uint32_t i = 0; i < pHeader->attributes.count; ++i)
+          {
+            if (pHeader->attributes.pDescriptors[i].typeInfo == vdkAttributeTypeInfo_uint8 && udStrEqual(pHeader->attributes.pDescriptors[i].name, "udClassification"))
+            {
+              uint8_t classificationID = *(uint8_t *)pAttributePtr;
+              pAttributePtr += 1;
+
+              const char *pClassificationName = vcSettingsUI_GetClassificationName(pProgramState, classificationID);
+
+              pProgramState->udModelNodeAttributes.Set("%s = '%s'", pHeader->attributes.pDescriptors[i].name, pClassificationName);
+              continue;
+            }
+
+            // Do Stuff
+            switch (pHeader->attributes.pDescriptors[i].typeInfo)
+            {
+            case vdkAttributeTypeInfo_uint8:
+              pProgramState->udModelNodeAttributes.Set("%s = %u", pHeader->attributes.pDescriptors[i].name, *(uint8_t *)pAttributePtr);
```

Comment:
Not a huge fan of unaligned reads, is it possible to avoid this? Perhaps a template that `memcpy`s into a temporary variable that is then returned?